components/network/samba/Makefile

```diff
@@ -34,7 +34,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		samba
 COMPONENT_VERSION=	4.9.4
-COMPONENT_REVISION=	1
+COMPONENT_REVISION=     0
```

Comment:
Please remove the whole line containing "COMPONENT_REVISION" in order to reset it.